lib/routes/content.js

```diff
@@ -103,14 +119,92 @@ const routes = application => {
     });
 
     /*
-     *  Post to Content Type
-     */
-    app.post(`/${config.content.base}/:type/${config.content.actions.save}`, multipartMiddleware, (req, res, next) => {
+     * @name Individual Content Type Edit Page
+     *
+     * @param {object} req - HTTP Request
+     * @param {object} res - HTTP Response
+     * @param {object} next - Express callback
+    */
+    app.get(`/${config.content.base}/:type/:id/${config.content.actions.edit}`, (req, res, next) => {
+      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
+
+      // need to merge errors for edit/revisions
+      // const errors = _.get(req.session, 'form.content.edit.errors', {});
+      let values = _.get(req.session, 'form.content.edit.content', {});
+      const data = {};
+
+      _.unset(req.session, 'form.content.edit');
+
+      values = utils.config(values);
+
+      if (type === false) {
+        _.set(req.session, '404', {
+          message: config.content.messages.missing.type.replace('%type', req.params.type),
+          safe: `/${config.content.base}`,
+        });
+
+        return next();
+      }
+
+      return database(`content-type--${type.id}`).where('id', req.params.id).then(rows => {
+        if (rows.length < 1) {
+          _.set(req.session, '404', {
+            message: config.content.messages.missing.id.replace('%type', req.params.type).replace('%id', req.params.id),
+            safe: `/${config.content.base}/${req.params.type}`,
+          });
+
+          return next();
+        }
+
+        const value = rows[0].value;
+
+        // mapping data from database
+        Object.keys(value).map(key => {
+          // if sunrise/sunset object is structured differently
+          if ((key === 'sunset-date') || (key === 'sunset-time') || (key === 'sunrise-date') || (key === 'sunrise-time')) {
+            data[key] = {
+              'value': value[key],
+            };
+          }
+          else {
+            const input = key.split('--');
```

Comment:
content-types [includes the split functionality](https://github.com/punchcard-cms/content-types/blob/master/lib/form/validate.js#L22-L50) in `validate` and supports repeatables. Maybe we should expose that instead of re-writing that functionality?